framework/core/js/src/forum/components/AccessTokensList.tsx

```diff
@@ -0,0 +1,182 @@
+import Component, { ComponentAttrs } from '../../common/Component';
+import type Mithril from 'mithril';
+import type AccessToken from '../../common/models/AccessToken';
+import app from '../app';
+import icon from '../../common/helpers/icon';
+import uaParser from 'ua-parser-js';
+import Button from '../../common/components/Button';
+import humanTime from '../../common/helpers/humanTime';
+import ItemList from '../../common/utils/ItemList';
+import DataSegment from '../../common/components/DataSegment';
+import extractText from '../../common/utils/extractText';
+import classList from '../../common/utils/classList';
+import Tooltip from '../../common/components/Tooltip';
```

Comment:
Just sorting the imports a bit

```suggestion
import app from '../app';
import Component, { ComponentAttrs } from '../../common/Component';
import icon from '../../common/helpers/icon';
import uaParser from 'ua-parser-js';
import Button from '../../common/components/Button';
import humanTime from '../../common/helpers/humanTime';
import ItemList from '../../common/utils/ItemList';
import DataSegment from '../../common/components/DataSegment';
import extractText from '../../common/utils/extractText';
import classList from '../../common/utils/classList';
import Tooltip from '../../common/components/Tooltip';
import type Mithril from 'mithril';
import type AccessToken from '../../common/models/AccessToken';
```